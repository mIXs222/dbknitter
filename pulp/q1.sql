SELECT name, age, num_cars
  FROM people
  JOIN (
    SELECT owner, COUNT(*) AS num_cars
      FROM cars
      GROUP BY owner
  ) AS cars_by_owner ON people.name = cars_by_owner.owner;