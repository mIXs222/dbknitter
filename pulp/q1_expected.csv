name,age,num_cars
"Vincent Vega",38,1
"Jules",41,1
"Mia",34,1
"Marsellus",45,5
"Mr. Wolf",56,1